eframe/src/web/backend.rs

```diff
@@ -367,18 +408,59 @@ impl AppRunner {
 
 pub type AppRunnerRef = Arc<Mutex<AppRunner>>;
 
+pub struct TargetEvent {
+    target: EventTarget,
+    event_name: String,
+    closure: Closure<dyn FnMut(web_sys::Event)>,
+}
+
+pub struct RepetitativeHandle {
```

Comment:
Its a feature. It can be used in future. For possible events in future. It was used for that that rerendering each each second event, which was removed for some reason.